### PR TITLE
[Ide] Let the references search be a child of the notebook

### PR DESCRIPTION
**Pull request by sevoku**
This is a partial fix for VSTS #750263, AtkCocoa needs to support the right focus order, including an Action widget.

### CODE REVIEW

**Comment by sevoku** [comment]
@monojenkins backport release-8.4

**Comment by sevoku** [comment]
@monojenkins backport release-8.4

**Comment by sevoku** [comment]
@monojenkins backport release-8.4

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Projects/SelectReferenceDialog.cs [hunk 1]
```diff
@@ -157,16 +158,21 @@ TreeIter AddPackageReference (ProjectReference refInfo)
 			}
 			return refTreeStore.AppendValues (txt, secondaryTxt, GetTypeText (refInfo), refInfo.Reference, refInfo, ImageService.GetIcon ("md-package", IconSize.Dnd));
 		}
-		
+
+		[DllImport ("libgtk-win32-2.0-0.dll", CallingConvention = CallingConvention.Cdecl)]
+		private static extern void gtk_notebook_set_action_widget (IntPtr notebook, IntPtr widget, int packType);
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Projects/SelectReferenceDialog.cs]: We could offer an extension for Gtk.Notebook with the implementation.

```csharp
class GtkWorkarounds
{
    private static extern void gtk_notebook_set_action_widget (IntPtr notebook, IntPtr widget, int packType);

    internal static void SetActionWidget (Gtk.Notebook notebook, Gtk.Widget widget, PackType packType)
    {
        gtk_notebook_set_action_widget (notebook.Handle, widget.Handle, (int)packType);
    }
}
```

And use that here.

**Pull request merged**